bower.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "react-copy-to-clipboard",
+  "version": "1.0.1",
+  "homepage": "https://github.com/nkbt/react-copy-to-clipboard",
+  "authors": [
+    "Nik Butenko <nik@butenko.me> (http://butenko.me/)"
+  ],
+  "description": "Copy-to-clipboard React component",
+  "main": "lib/index.js",
```

Comment:
Yeah, I've got the point... Will publish on bower soon. Had to fix some important things in https://github.com/in-flux/component-router/ today. So tomorrow will publish this one. And hopefully in next couple of days will be able to automate bower publishing for other components.